src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/lambda_codebase/enable_cross_account_access.py

```diff
@@ -64,10 +64,10 @@ def lambda_handler(event, _):
                         'adf-cloudformation-deployment-role'
                         ), 'base_cfn_role'
                 )
-                LOGGER.debug("Role has bee assumed for %s", account_id)
+                LOGGER.debug("Role has been assumed for %s", account_id)
                 update_iam(role, s3_bucket, kms_key_arn, target_role_policies)
-            except ClientError:
-                LOGGER.debug("%s not yet configured, continuing", account_id)
+            except ClientError as err:
+                LOGGER.debug("%s could not be assumed (%s), continuing", account_id, err)
```

Comment:
The Python logger has support for capturing the stacktrace too. This will help debug in the future.
You can add it by setting `exc_info=True`, like so:

```
LOGGER.debug("%s could not be assumed (%s), continuing", account_id, err, exc_info=True)
```

Reply:
fixed! thanks!